### PR TITLE
Business entities

### PR DESCRIPTION
**Pull request by PaulReeve717**
Added the ShoppingCart entity

### CODE REVIEW

File: ochap-backend/src/main/resources/application.properties [hunk 1]
```diff
@@ -2,3 +2,4 @@ spring.jpa.hibernate.ddl-auto=update
 spring.datasource.url=jdbc:postgresql://localhost:5432/ochap-backend
 spring.datasource.username=ochap-backend-user
 spring.datasource.password=ochap-backend-integration-tests-password
+spring.jpa.show-sql=true
```
Comment (roosemberth) [inline ochap-backend/src/main/resources/application.properties:5]: Is this change intended?

**Review by roosemberth — CHANGES_REQUESTED** [review]

File: ochap-backend/src/main/resources/application.properties (hunk 1 above)
Comment (PaulReeve717) [reply ochap-backend/src/main/resources/application.properties]: Nope, I'll remove it

**Review by roosemberth — APPROVED** [review]
Thanks for your contribution!
